src/components/Charts/ChataHistogram/ChataHistogram.js

```diff
@@ -295,17 +293,17 @@ export default class ChataHistogram extends React.Component {
             linearAxis='y'
             yGridLines
           >
-            {this.axisIsNumerical() ? (
-              <HistogramColumns
-                {...this.props}
-                xScale={this.xScale}
-                yScale={this.yScale}
-                buckets={this.buckets}
-                bins={this.bins}
-              />
-            ) : (
+            {/* {this.axisIsNumerical() ? ( */}
```

Comment:
is this comment necessary not to be deleted?
Or if you want to comment a TO DO on this kind of comments through the files?